bitnami/contour/values.yaml

```
@@ -44,18 +44,18 @@ configInline:
   # disable ingressroute permitInsecure field
   disablePermitInsecure: false
   tls:
-  #   minimum TLS version that Contour will negotiate
-  #   minimum-protocol-version: "1.1"
-  # Defines the Kubernetes name/namespace matching a secret to use
-  # as the fallback certificate when requests which don't match the
-  # SNI defined for a vhost.
+    #   minimum TLS version that Contour will negotiate
+    #   minimum-protocol-version: "1.1"
```

In cases like these (and throughout the rest of the files), I think we should be removing the spaces before the object, i.e.:

```suggestion
    # minimum TLS version that Contour will negotiate
    # minimum-protocol-version: "1.1"
```

Please make sure to review all occurrences like these, as I've seen many.

---

done!